src/main/java/ab/common/lib/register/RecipeListAB.java

```diff
@@ -1,5 +1,8 @@
 package ab.common.lib.register;
 
+import static gregtech.api.enums.Mods.ForbiddenMagic;
+import static gregtech.api.util.GT_ModHandler.getModItem;
+
```

Comment:
this wont work, as GT is yeeted from the deps

Reply:
This was the code that @chochem give to me to solve a variable, idk if it will work without that

Reply:
copying code without understanding it is a very bad practice.

Reply:
that was before we started talking about removing gt as a dep obviously